wled00/xml.cpp

```diff
@@ -152,6 +159,7 @@ void getSettingsJS(byte subPage, Print& settingsScript)
   DEBUG_PRINTF_P(PSTR("settings resp %u\n"), (unsigned)subPage);
 
   if (subPage <0 || subPage >10) return;
+  char nS[32];
```

Comment:
Did you find this reduced the overall function stack usage?  Otherwise my preference runs to the more narrowly scoped variables.

Reply:
just removes annoying "unused variable" warning